src/main/java/com/spartasystems/holdmail/domain/Message.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright 2016 Sparta Systems, Inc
+ * Copyright 2016 - 2017 Sparta Systems, Inc
```

Comment:
We should take a look.  NOTICE file handles the inception year part, so we don't really need the inception year in the header (btw would be a good time to update NOTICES)